### DISCUSSION TITLE
Filebeat Module Intake

### DISCUSSION THREAD

**Discussion by Tryntius** [Security-Onion-Solutions/securityonion#5701] [Unsupported Versions]
I have configured a single node deployment currently intaking syslog on port 514 without issue.  I'm new to security onion, and I have been trying to follow the workflow for the filebeat module: https://docs.securityonion.net/en/latest/filebeat.html

I have used so-firewall to create a port group for 9004 for my cisco device and my PC.  I am able to telnet successfully.  iptables verifies my rules are correct.

I have appended this to the bottom of my minion pillar:
filebeat:
  modules:
    cisco:
      ftd:
        enabled: true
        var.input: udp
        var.syslog_host: 0.0.0.0
        var.syslog_port: 9004

I had to leave out "third_party_filebeat:" as that line would crash filebeat within two minutes of a restart.  I have also enabled debug logging on filebeat, but I couldn't find anything helpful in the logs (yet).  

When my FTD sends logs to 514, everything shows up, but the parsing is terrible.  When I try to send them to 9004, nothing appears.  What step did I miss?  My understanding was with modules the work flow is: enable the firewall rules, enable the module intake on a specific port, switch syslog output to said port.

**Comment by weslambert** [comment]
If you are intending to have the Cisco syslog parsed, I would send to port 9004, like you've mentioned.  You need the `third_party_filebeat` line to ensure that the configuration is applied and the module is enabled.  Have you tried tailing the Filebeat log in /opt/so/log/filebeat while it's running to get an idea of the issue?
